### ISSUE TITLE
Cannot open deeplinker with https protocol

### ISSUE THREAD

**Issue by idlleF** [ionic-team/ionic-plugin-deeplinks#170]
```
 $ionicPlatform.ready(function () {
        $cordovaDeeplinks.route({
            '/profile/:id': {
                target: 'profile'
            }
        }).subscribe(function (match) {
            $timeout(function () {
                $state.go(match.$route.parent, match.$args);
                $timeout(function () {
                    $state.go(match.$route.target, match.$args);
                }, 1000);
            }, 100);
        }, function (nomatch) {
            console.log('No match', nomatch);
        });
    });
```

I've installed the plugin following the guide. When i try to access my app from simulator browser (on iOS) dont even try to open the app, but with custom url scheme it works

Anyone  have same problem ?

**Comment by pavlo-liapin** [comment]
Same issue for me

**Comment by pavlo-liapin** [comment]
@idlleF https://github.com/ionic-team/ionic-plugin-deeplinks/issues/162 should help, Associated domains must be enabled manually in XCode or via `config.xml`

**Comment by idlleF** [comment]
@pavlo-liapin , yes. Besides that you must set correct json file in your "waiting" https domain, but that information missing in plugin documentation.

**Comment by ViVer-7** [comment]
I have a similar problem:
https://stackoverflow.com/q/54255460/3496701

Hopefully anybody can help me...

**Comment by JerryBels** [comment]
Having the same issue. 

Added the config.xml stuff as found in https://github.com/ionic-team/ionic-plugin-deeplinks/issues/162 and https://github.com/ionic-team/ionic-plugin-deeplinks/issues/154#issuecomment-432174320

Added the file in `https://mysite.com/.well-known/apple-app-site-association`

Content of the file : 

```
{
  "applinks": {
    "apps": [],
    "details": [
      {
        "appID": “MYTEAMID.com.mysite",
        "paths": [
          "*"
        ]
      }
    ]
  }
}
```

Associated Domains is active on developers.apple.com account and the domain shows up on Xcode.

And still, when launching the app on the simulator, if opening an https url `https://mysite.com/item/itemid` that should trigger the app, the browsers opens right away...
